jolt-core/src/test/java/com/bazaarvoice/jolt/utils/JoltUtilsTest.java

```diff
@@ -161,6 +161,52 @@ public void runFixtureTests() throws IOException {
         }
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void replaceRecursive() throws IOException {
+
+        String testFixture = "/json/utils/joltUtils-replaceRecursive.json";
+
+        List<Map<String, Object>> tests = (List<Map<String, Object>>) JsonUtils.classpathToObject( testFixture );
+
+        for ( Map<String,Object> testUnit : tests ) {
+            Object data = testUnit.get( "input" );
+            Map<String,Object> toReplace = (Map<String, Object>)testUnit.get( "replace" );
+            Object expected = testUnit.get( "expected" );
+
+            for(String key : toReplace.keySet()) {
```

Comment:
This is clever in that it is using the same test fixture to test two methods, but it would be simpler to understand if this test only changed one key / had a separate test fixture where only one key got replaced/updated.

Reply:
Now I removed this test and just test the replace value, since now we can give a path for each key to be replaced and a set of values that can be replaced.